dotnet/src/SemanticKernel.Abstractions/Http/HttpHeaderConstant.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+
+namespace Microsoft.SemanticKernel.Http;
+
+/// <summary>
+/// Provides HTTP header names and values for common purposes.
+/// </summary>
+public static class HttpHeaderConstant
+{
+    /// <summary>
+    /// Provides HTTP header names for common purposes.
+    /// </summary>
+    public static class Names
+    {
+        /// <summary>
+        /// HTTP header name to use to include the Semantic Kernel package version in all HTTP requests issued by Semantic Kernel.
+        /// </summary>
+        public static string SemanticKernelVersion => "Semantic-Kernel-Version";
+    }
+
+    /// <summary>
+    /// Provides HTTP header values for common purposes.
+    /// </summary>
+    public static class Values
+    {
+        /// <summary>
+        /// User agent string to use for all HTTP requests issued by Semantic Kernel.
+        /// </summary>
+        public static string UserAgent => s_userAgent;
+
+        /// <summary>
+        /// Gets the version of the <see cref="System.Reflection.Assembly"/> in which the specific type is declared.
+        /// </summary>
+        /// <param name="type">Type for which the assembly version is returned.</param>
+        public static string GetAssemblyVersion(Type type)
```

Comment:
We probably don't want to include this in the publicly exposed bits.  It's not really related to the user agent and not required publicly.